exercises/julia-1-a/.meta/example.jl

```diff
@@ -0,0 +1,36 @@
+function gamestate(bs)
```

Comment:
This is a rather naïve solution and probably not very performant.

Reply:
The existing exercises all have three example solutions, to demonstrate that there are indeed three possible implementations. They might be used in the second part of the exercise too, as starting points.